CMakeLists.txt

```
@@ -509,6 +509,14 @@ endif()
 if(OPENMC_USE_DAGMC)
   target_compile_definitions(libopenmc PRIVATE DAGMC)
   target_link_libraries(libopenmc dagmc-shared)
+
+  if(OPENMC_USE_UWUW)
+    target_compile_definitions(libopenmc PRIVATE OPENMC_UWUW)
+    target_link_libraries(libopenmc uwuw-shared)
+  endif()
+elseif(OPENMC_USE_UWUW)
+  set(OPENMC_USE_UWUW OFF)
+  message(WARNING "UWUW is enabled but DAGMC was not found. Disabling.")
```

I think we should go ahead and make this an error so it doesn't get lost by the user in the CMake output. If DAGMC isn't enabled there's really no point in enabling UWUW right now and I'd consider it an invalid configuration.

```suggestion
  message(FATAL_ERROR "DAGMC is required for use of UWUW materials.")
```

I'll note here that this doesn't have to be the case and that someday I'd like to refactor the code to support for UWUW materials from DAGMC geometry in OpenMC and separate the usage of these libraries.